packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -169,6 +169,8 @@ export const integrationsHandler = async ({ ctx, input }: IntegrationsOptions) =
         invalidCredentialIds,
         teams,
         isInstalled: !!userCredentialIds.length || !!teams.length || app.isGlobal,
+        // FIXME: remove hardcoding and add per-app validation
+        isSetup: !!(credential?.key as { account_id: string })?.account_id || app.slug !== "alby",
```

Comment:
We can leave this for later I think. The idea we can probably validate using zod schemas.

Reply:
updated comment, thanks